tests/components/onkyo/conftest.py

```diff
@@ -1,23 +1,14 @@
 """Configure tests for the Onkyo integration."""
 
-from collections.abc import Generator
-from unittest.mock import AsyncMock, patch
+from unittest.mock import patch
 
 import pytest
 
 from homeassistant.components.onkyo.const import DOMAIN
 
-from tests.common import MockConfigEntry
-
+from . import create_connection
 
-@pytest.fixture
-def mock_setup_entry() -> Generator[AsyncMock]:
```

Comment:
@arturpragacz I deleted this mock_setup_entry fixture method in here, it didn't appear to be used anywhere, and no tests fail when I remove it.

Reply:
Yes, that is very possible.

The tests are a bit of a mess right now. They were introduced by #117319. That PR was in progress for a very long time, and in the end we just decided to bite the bullet, merge it in, and clean up the tests later.

The author of that PR still intends to do it, just hasn't had the time yet.